### PR TITLE
Add section to the docs about topology labels

### PR DESCRIPTION
**Pull request by dstathis**
## Issue
#118 

## Solution
Add documentation.

## Testing Instructions
None! Just give it a read.

## Release Notes
Added documentation.

### CODE REVIEW

File: lib/charms/loki_k8s/v0/loki_push_api.py [hunk 1]
```diff
@@ -218,6 +218,12 @@ def __init__(self, *args):
 
 The consumer charm can then choose to update its configuration in both situations.
 
+Note that when LokiPushApiConsumer is used, logs will not have any labels automatically added. In
+order to better integrate with the Canonical Observability Stack, you may want to configure your
```
Comment (sed-i) [inline lib/charms/loki_k8s/v0/loki_push_api.py:222]:
```suggestion
Note that LokiPushApiConsumer does not add any labels automatically on its own. In
order to better integrate with the Canonical Observability Stack, you may want to configure your
```

File: lib/charms/loki_k8s/v0/loki_push_api.py [hunk 2]
```diff
@@ -218,6 +218,12 @@ def __init__(self, *args):
...
+software to add Juju topology labels. The
```
Comment (sed-i) [inline lib/charms/loki_k8s/v0/loki_push_api.py:223]:
```suggestion
charm to add Juju topology labels. The
```

File: lib/charms/loki_k8s/v0/loki_push_api.py [hunk 3]
```diff
@@ -218,6 +218,12 @@ def __init__(self, *args):
...
+[observability-libs](https://charmhub.io/observability-libs) library can be used to get topology
+labels in charm code.
```
Comment (sed-i) [inline lib/charms/loki_k8s/v0/loki_push_api.py:225]: I think a concrete example here could go a long way.

Comment (dstathis) [reply lib/charms/loki_k8s/v0/loki_push_api.py:215]: As discussed in our meeting, an example could become out of date and there is a link to the docs here.

**Review by Abuelodelanada — DISMISSED** [review]

File: lib/charms/loki_k8s/v0/loki_push_api.py (hunk 3 above)
Comment (sed-i) [reply lib/charms/loki_k8s/v0/loki_push_api.py:215]:
```suggestion
[observability-libs](https://charmhub.io/observability-libs) library can be used to get topology
labels in charm code. See :func:`LogProxyConsumer._scrape_configs` for an example of how
to do this with promtail.
```

**Review by sed-i — DISMISSED** [review]

**Review by rbarry82 — APPROVED** [review]

**Review by Abuelodelanada — APPROVED** [review]

**Pull request merged**